.github/workflows/main.yml

```diff
@@ -0,0 +1,108 @@
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          # fetch entire history to compute diffs between jobs
+          fetch-depth: 0
+
+      - uses: ./
+        id: mcode-action
+        continue-on-error: true
+        with:
+          mayhem-url: https://demo.forallsecure.com
+          mayhem-token: ${{ secrets.MAYHEM_TOKEN }}
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          sarif-output: out/sarif/
+          args: --image forallsecure/lighttpd:vulnerable --file __tests__/Mayhemfile --duration 60
+          
+      - name: Upload SARIF file(s)
+        uses: github/codeql-action/upload-sarif@v2
+        with:
+          sarif_file: out/sarif
+
+      - name: Print runId (${{ steps.mcode-action.outputs.runId }})
+        run: |
+          echo "The run id was: ${{ steps.mcode-action.outputs.runId }}"
+
+  test-all-outputs: # make sure the action works on a clean machine without building
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          # fetch entire history to compute diffs between jobs
+          fetch-depth: 0
+
+      - uses: ./
+        id: mcode-action
+        continue-on-error: true
+        with:
+          mayhem-url: https://demo.forallsecure.com
+          mayhem-token: ${{ secrets.MAYHEM_TOKEN }}
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          junit-output: out/junit/
+          sarif-output: out/sarif/
+          coverage-output: out/coverage/
+          args: --image forallsecure/lighttpd:vulnerable --file __tests__/Mayhemfile --duration 60
+
+      - name: Archive Coverage report
+        uses: actions/upload-artifact@v3
+        with:
+          name: coverage-report
+          path: out/coverage/
+
+      - name: Archive JUnit results
+        uses: actions/upload-artifact@v3
+        with:
+          name: mcode-junit
+          path: out/junit/
+          
+      - name: Upload SARIF file(s)
+        uses: github/codeql-action/upload-sarif@v2
+        with:
+          sarif_file: out/sarif
+
+      - name: Print runId (${{ steps.mcode-action.outputs.runId }})
+        run: |
+          echo "The run id was: ${{ steps.mcode-action.outputs.runId }}"
+
+  test-no-outputs: # make sure the action works on a clean machine without building
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          # fetch entire history to compute diffs between jobs
+          fetch-depth: 0
+
+      - uses: ./
+        id: mcode-action
+        continue-on-error: true
+        with:
+          mayhem-url: https://demo.forallsecure.com
+          mayhem-token: ${{ secrets.MAYHEM_TOKEN }}
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          args: --image forallsecure/lighttpd:vulnerable --file __tests__/Mayhemfile --duration 60
+
+      - name: Print runId (${{ steps.mcode-action.outputs.runId }})
+        run: |
+          echo "The run id was: ${{ steps.mcode-action.outputs.runId }}"
```

Comment:
Ditto to the more specific comment

Reply:
[1](https://github.com/ForAllSecure/mcode-action/actions/runs/4317369492/jobs/7534353438#step:4:1)
Run [ -z "mcode-action/lighttpd/29" ] && echo "runId was blank!" && exit 1;